todocli/cli.py

```diff
@@ -111,14 +112,14 @@ def print_list(item_list, print_line_nums):
 
 
 def ls(args):
-    lists = todo_api.query_lists()
-    lists_names = [task_list["displayName"] for task_list in lists]
+    lists = [TodoList(x) for x in todo_api.query_lists()]
+    lists_names = [l.display_name for l in lists]
     print_list(lists_names, args.display_linenums)
 
 
 def lst(args):
-    tasks = todo_api.query_tasks(args.list_name)
-    tasks_titles = [x["title"] for x in tasks]
+    tasks = [Task(x) for x in todo_api.query_tasks(args.list_name)]
```

Comment:
Same comment as above. I really think classes should be instantiated in `todo_api`.